pubspec.yaml

```diff
@@ -0,0 +1,35 @@
+name: app_buy_sell
+description: "A new Flutter project."
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+version: 1.0.0+1
+
+environment:
+  sdk: '>=3.2.4 <4.0.0'
+dependencies:
+  flutter:
+    sdk: flutter
+  cupertino_icons: ^1.0.2
```

Comment:
Please use riverpod and riverpod generator for state management.
We won't use rxdart, koin, koin_flutter, and provider.
Besides, could you follow this project structure?

https://codewithandrea.com/articles/flutter-project-structure/#how-to-do-feature-first-the-right-way